src/bin/vip-dev-env-import-sql.js

```diff
@@ -54,6 +54,7 @@ command( {
 		const slug = getEnvironmentName( opt );
 
 		const lando = await bootstrapLando();
+		lando.events.constructor.prototype.setMaxListeners( 100 );
```

Comment:
AsyncEvents [hardcodes](https://github.com/Automattic/lando-cli/blob/93b9137e25/lib/events.js#L149) the number of event listeners to 64.

We need a higher value because we issue several calls to exec().